flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/SourceOperatorStreamTaskTest.java

```diff
@@ -74,6 +84,46 @@
     private static final OperatorID OPERATOR_ID = new OperatorID();
     private static final int NUM_RECORDS = 10;
 
+    @Test(timeout = 60_000)
```

Comment:
I think the consensus in the ML, was that we don't want to add timeouts anymore in JUnit tests.